recipes/trafaret/LICENSE.txt

```diff
@@ -0,0 +1,25 @@
+Copyright (c) 2013, 2014, Mikhail Krivushin and Yehor Nazarkin.
```

Comment:
I don't think this file is needed if the license is included in the package (we should be able to pull it from the tarball). If it isn't please follow the steps for [packaging a license](https://conda-forge.org/docs/meta.html#packaging-the-license-manually).